app/front/scripts/controllers/main.js

```diff
@@ -267,6 +278,8 @@
             }
           }
 
+          $scope.isEmbeded = $window.isEmbeded;
```

Comment:
@akariv  no, Levko was changing this controller too in his commits, so i do not do the same work and to avoid some conflicts